config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/studios", to: "studios#index"
+
+  get "/movies/:id", to: "movies#show"
+  post "/movies/:id", to: "movie_actors#create"
```

Comment:
Since this is going to `movie_actors#create`, I'd also call the route:
`post "/movie_actors", to: "movie_actors#create"` which is more restful.